mmv1/templates/terraform/pre_create/kms_autokey_config_folder.go.erb

```diff
@@ -0,0 +1 @@
+url = strings.Replace(url, "folders/folders/", "folders/", 1)
```

Comment:
```suggestion
url = strings.Replace(url, "folders/folders/", "folders/", 1)
folderValue := d.Get("folder").(string)
folderValue = strings.Replace(folderValue, "folders/", "", 1)
d.Set("folder", folderValue)
```

Restoring this to as it was before https://github.com/GoogleCloudPlatform/magic-modules/pull/11413#discussion_r1717174182